src/casts/interpret-casts-with-hyper-coercions.rkt

```diff
@@ -1,21 +1,14 @@
 #lang typed/racket/base
 #|
 Pass: interpret casts with hyper-coercions
-Author: Andre Kuhlenshmidt (akuhlens@indiana.edu)                            
```

Comment:
This file is changed with almost the same changes in this commit and in the previous one, why?